### DISCUSSION TITLE
Can someone guide me? I am completely new to this.

### DISCUSSION THREAD

**Discussion by RpgRiser** [acheong08/ChatGPT#408] [Q&A]
Hey there, I was looking on Reddit how to access ChatGPT as it's not available in my country, and I found a link to this. Issue is, I have no idea how to do most of what's explained here, I know nothing about programming and python. I installed python, downloaded the file, installed it through command line, it downloaded some stuff and now I have no idea what's the next step. Could someone please guide me? I'm on Windows 10.

**Comment by acheong08** [comment]
It's not possible to access ChatGPT without an account.

**Reply by RpgRiser** [reply]
Ah well, my bad, and thanks for the reply. Guess I'll just have to find someone willing to share their account ;-;

**Comment by DiamondDemon669** [comment]
Try a VPN

**Reply by acheong08** [reply]
Still needs a phone number
